src/runtime_src/core/edge/drm/zocl/common/zocl_sysfs.c

```diff
@@ -600,7 +600,7 @@ static ssize_t read_xclbin_full(struct file *filp, struct kobject *kobj,
 	read_lock(&zdev->attr_rwlock);
 
 	// Only read slot 0's xclbin - TODO: extend to multi-slot 
-	zocl_slot = zdev->pr_slot[0];
+	zocl_slot = zdev->pr_slot[1];
```

Comment:
This index should come as an input. 
Think about, if two or more PS XCLBINs are  loaded

Reply:
this is a dead code, sysfs functionality is being used anymore